avm/res/machine-learning-services/registry/main.bicep

```
@@ -0,0 +1,337 @@
+metadata name = 'Azure ML Registry'
+metadata description = 'This module deploys an Azure Machine Learning Registry.'
+
+@description('Required. Name of the Azure ML Registry.')
+@minLength(5)
+@maxLength(50)
+param name string
+
+@description('Optional. Location for the Azure ML Registry.')
+param location string = resourceGroup().location
+
+@description('Optional. Additional locations for the Azure ML Registry.')
+param locations array = []
```

```suggestion
param locations string[] = []
```
I guess that's even more precise, right?

---

right!